src/util/fpscounter.rs

```diff
@@ -0,0 +1,41 @@
+use ::time;
+use ::circular_buffer;
+use std::collections::VecDeque;
+use amethyst::ecs::System;
+
+///The FPSCounter resource needed by the FPSCounterSystem
+///Add it to your resources to be able to use the FPSCounterSystem
+///world.add_resource(FPSCounter::new(20));
+///Usage example:
+///Get the FPSCounter resource.
+///println!("Cur FPS: {}, Sampled: {}",counter.frame_fps(),counter.sampled_fps());
+pub struct FPSCounter{
```

Comment:
Move this into src/ecs/util/resources in order to conform with the rest of the crate structure.